### PR TITLE
Add React to an Existing Project

### PR DESCRIPTION
**Pull request by Rados51**
Resolve https://github.com/reactjs/cs.react.dev/issues/8

### CODE REVIEW

File: src/content/learn/add-react-to-an-existing-project.md [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
 
 <Note>
 
-**You need to install [Node.js](https://nodejs.org/en/) for local development.** Although you can [try React](/learn/installation#try-react) online or with a simple HTML page, realistically most JavaScript tooling you'll want to use for development requires Node.js.
+**Pro vývoj v lokálním prostředí je nutné mít nainstalovaný [Node.js](https://nodejs.org/en/).** I když máte možnost si [vyzkoušet React](/learn/installation#try-react) online nebo na jednoduché HTML stránce, realita je taková, že většina JavaScriptových nástrojů, které budete pravděpodobně chtít použít pro vývoj, vyžaduje Node.js.
 
 </Note>
 
-## Using React for an entire subroute of your existing website {/*using-react-for-an-entire-subroute-of-your-existing-website*/}
+## Implementace Reactu do adresářové cesty vaší stávající webové stránky {/*using-react-for-an-entire-subroute-of-your-existing-website*/}
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:17]: I think it is hard to understand. It should be clear that you can use React only on part of the website. I think the "adresářové cesty" is misleading.
```suggestion
## Implementace Reactu do adresářové cesty vaší stávající webové stránky {/*using-react-for-an-entire-subroute-of-your-existing-website*/}
```

File: src/content/learn/add-react-to-an-existing-project.md [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
 
-Let's say you have an existing web app at `example.com` built with another server technology (like Rails), and you want to implement all routes starting with `example.com/some-app/` fully with React.
+Představme si, že máte existující webovou aplikaci na adrese `domena.cz`, která byla vytvořena za pomocí jiné serverové technologie (např. Rails) a chcete všechny cesty začínající na `domena.cz/obchod/` plně implementovat pomocí Reactu.
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:19]: Just some minor suggestions. 
```suggestion
Představme si, že máte existující webovou aplikaci na adrese `domena.cz`, která byla vytvořena za pomocí jiné serverové technologie (např. Rails) a chcete, aby všechny cesty začínající s `domena.cz/obchod/` byly plně implementované pomocí Reactu.
```

Comment (pchalupa) [reply src/content/learn/add-react-to-an-existing-project.md:19]: I think the `example.com` should not be translated.

File: src/content/learn/add-react-to-an-existing-project.md [hunk 3]
```diff
@@ -1,59 +1,59 @@
...
 
-Here's how we recommend to set it up:
+Doporučený postup, jak to nastavit:
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:21]: Maybe?
```suggestion
Zde je náš doporučený postup:
```

File: src/content/learn/add-react-to-an-existing-project.md [hunk 4]
```diff
@@ -1,59 +1,59 @@
...
-3. **Configure your server or a proxy** so that all requests under `/some-app/` are handled by your React app.
+1. **Vytvořte část aplikace** pomocí některého z [frameworků založených na Reactu](/learn/start-a-new-react-project).
+2. **Nastavte `/obchod` jako *základní cestu*** v konfiguraci vašeho frameworku (zde je návod pro: [Next.js](https://nextjs.org/docs/api-reference/next.config.js/basepath) a [Gatsby](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/path-prefix/)).
+3. **Nakonfigurujte svůj server nebo proxy server** tak, aby všechny požadavky na cestu `/obchod/` zpracovávala vaše aplikace v Reactu.
 
-This ensures the React part of your app can [benefit from the best practices](/learn/start-a-new-react-project#can-i-use-react-without-a-framework) baked into those frameworks.
+Takto zajistíte, že část vaší aplikace napsaná Reactu bude moci využít [osvědčené postupy a praktiky](/learn/start-a-new-react-project#can-i-use-react-without-a-framework), které jsou součástí těchto frameworků.
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:27]: A minor suggestion.
```suggestion
Tímto zajistíte, že část vaší aplikace napsaná Reactu bude využívat [osvědčené postupy a praktiky](/learn/start-a-new-react-project#can-i-use-react-without-a-framework), které jsou součástí těchto frameworků.
```

File: src/content/learn/add-react-to-an-existing-project.md [hunk 5]
```diff
@@ -1,59 +1,59 @@
...
 
-Many React-based frameworks are full-stack and let your React app take advantage of the server. However, you can use the same approach even if you can't or don't want to run JavaScript on the server. In that case, serve the HTML/CSS/JS export ([`next export` output](https://nextjs.org/docs/advanced-features/static-html-export) for Next.js, default for Gatsby) at `/some-app/` instead.
+Mnoho frameworků založených na Reactu je full-stack, což umožňuje vaší React aplikaci využít server. Tento přístup můžete uplatnit i v případě, kdy nemůžete nebo nechcete na serveru používat JavaScript. V takové situaci místo toho naservírujte export HTML/CSS/JS ([output v `next export`](https://nextjs.org/docs/advanced-features/static-html-export) pro Next.js, default pro Gatsby) na adresářové cestě `/obchod/`.
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:29]: The word `naservírujete` seems strange to me. What about "zpřístupnite".

File: src/content/learn/add-react-to-an-existing-project.md [hunk 6]
```diff
@@ -1,59 +1,59 @@
...
 
-## Using React for a part of your existing page {/*using-react-for-a-part-of-your-existing-page*/}
+## Implementace Reactu do konkrétní části vaší stávající webové stránky {/*using-react-for-a-part-of-your-existing-page*/}
 
-Let's say you have an existing page built with another technology (either a server one like Rails, or a client one like Backbone), and you want to render interactive React components somewhere on that page. That's a common way to integrate React--in fact, it's how most React usage looked at Meta for many years!
+Předpokládejme, že máte stávající stránku, která byla vytvořena s využitím jiné technologie, ať už serverové (např. Rails) nebo klientské (jako je Backbone), a chcete na ní implementovat interaktivní komponenty React. Tento postup je běžnou formou integrace Reactu. Ve skutečnosti se React takto používal řadu let v Metě!
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:33]: Is it right to use the word `Metě`?

File: src/content/learn/add-react-to-an-existing-project.md [hunk 7]
```diff
@@ -62,60 +62,60 @@ Then add these lines of code at the top of your main JavaScript file (it might b
...
 
-Instead, you probably want to render your React components in specific places in your HTML. Open your HTML page (or the server templates that generate it) and add a unique [`id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) attribute to any tag, for example:
+Spíše než vykreslovat celou stránku pomocí Reactu, budete pravděpodobně chtít vykreslit React komponenty na specifických místech v HTML. Otevřete svou HTML stránku (nebo serverové šablony, které ji generují) a přidejte například unikátní atribut [`id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) k libovolnému tagu:
 
 ```html
-<!-- ... somewhere in your html ... -->
+<!-- ... někde v html ... -->
 <nav id="navigation"></nav>
-<!-- ... more html ... -->
+<!-- ... další html ... -->
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:115]: Maybe?
```suggestion
<!-- ... více html ... -->
```

**Review by pchalupa — CHANGES_REQUESTED** [review]
I left some suggestions, but overall looks great to me.

File: src/content/learn/add-react-to-an-existing-project.md [hunk 8]
```diff
@@ -1,59 +1,59 @@
 ---
-title: Add React to an Existing Project
+title: Přidání Reactu do existujícího projektu
 ---
 
 <Intro>
 
-If you want to add some interactivity to your existing project, you don't have to rewrite it in React. Add React to your existing stack, and render interactive React components anywhere.
+Pokud máte v plánu přidat interaktivitu do svého existujícího projektu, není nutné celý projekt přepracovat do Reactu. Můžete jednoduše začlenit React mezi již použité technologie a umístit interaktivní React komponenty, kamkoliv je potřeba.
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:7]: I see great improvement there! Maybe the "do svého existujícího projektu" sounds a bit odd. What about "do existujícího projektu"?

File: src/content/learn/add-react-to-an-existing-project.md [hunk 9]
```diff
@@ -62,60 +62,60 @@ Then add these lines of code at the top of your main JavaScript file (it might b
...
 
-// Clear the existing HTML content
+// Nahradí existující obsah HTML
 document.body.innerHTML = '<div id="app"></div>';
 
-// Render your React component instead
+// Do něj nyní vykreslíme React komponentu
 const root = createRoot(document.getElementById('app'));
-root.render(<h1>Hello, world</h1>);
+root.render(<h1>Ahoj světe</h1>);
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:78]: Do we want to translate a text inside the code? We want to translate comments, but I am not sure about this line.

File: src/content/learn/add-react-to-an-existing-project.md [hunk 10]
```diff
@@ -62,60 +62,60 @@ Then add these lines of code at the top of your main JavaScript file (it might b
...
 ```
 
 </Sandpack>
 
-If the entire content of your page was replaced by a "Hello, world!", everything worked! Keep reading.
+Pokud byl celý obsah stránky nahrazen nápisem "Ahoj světe", vše funguje správně! Čtěte dál.
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:83]: Just an idea about the form of this sentence.
```suggestion
Pokud byl celý obsah stránky nahrazen nápisem "Ahoj světe", vše funguje správně! Pokračujte ve čtení.
```

File: src/content/learn/add-react-to-an-existing-project.md [hunk 11]
```diff
@@ -62,60 +62,60 @@ Then add these lines of code at the top of your main JavaScript file (it might b
...
 
-// Render your React component instead
+// Do něj nyní vykreslíme React komponentu
 const root = createRoot(document.getElementById('app'));
-root.render(<h1>Hello, world</h1>);
+root.render(<h1>Ahoj světe</h1>);
 ```
 
-Of course, you don't actually want to clear the existing HTML content!
+Samozřejmě, v reálném scénáři nebudete chtít existující obsah HTML vymazat!
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:106]: Maybe different form of the sentence. The "reálném scrénáři" sounds a bit off tome. Maybe "reálném případě" could also fit.
```suggestion
Samozřejmě, v reálu nebudete chtít existující obsah HTML vymazat!
```

**Review by pchalupa — CHANGES_REQUESTED** [review]
My final comments. I saw a great improvement! 👏

File: src/content/learn/add-react-to-an-existing-project.md (hunk 10 above)
Comment (Rados51) [reply src/content/learn/add-react-to-an-existing-project.md:83]: Much better, thanks!

File: src/content/learn/add-react-to-an-existing-project.md (hunk 9 above)
Comment (Rados51) [reply src/content/learn/add-react-to-an-existing-project.md]: We can decide on this later on. The Spanish translation uses `¡Hola, mundo!`, but we can keep `Hello, world`. I am okay with both.

**Review by pchalupa — DISMISSED** [review]
Good job! 👍

File: src/content/learn/add-react-to-an-existing-project.md [hunk 12]
```diff
@@ -62,60 +62,60 @@ Then add these lines of code at the top of your main JavaScript file (it might b
...
 ```
 
-Of course, you don't actually want to clear the existing HTML content!
+Samozřejmě, v reálném případě nebudete chtít existující obsah HTML vymazat!
 
-Delete this code.
+Tento kód odstraňte.
 
-Instead, you probably want to render your React components in specific places in your HTML. Open your HTML page (or the server templates that generate it) and add a unique [`id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) attribute to any tag, for example:
+Spíše než vykreslovat celou stránku pomocí Reactu, budete chtít vykreslit React komponenty na specifických místech v HTML. Otevřete svou HTML stránku (nebo serverové šablony, které ji generují) a přidejte například unikátní atribut [`id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) k libovolnému tagu:
```
Comment (pchalupa) [inline src/content/learn/add-react-to-an-existing-project.md:110]: I am very open to discussion, we can translate render to "vykreslit" but also to "rendrovat", which can fit better.

**Review by pchalupa — APPROVED** [review]

**Pull request merged**